packages/browser/src/plugins/remote-loader/index.ts

```diff
@@ -1,14 +1,25 @@
-import type { Integrations } from '../../core/events/interfaces'
+import type { Integrations, SegmentEvent } from '../../core/events/interfaces'
 import { LegacySettings } from '../../browser'
 import { JSONObject, JSONValue } from '../../core/events'
 import { Plugin } from '../../core/plugin'
 import { asPromise } from '../../lib/as-promise'
 import { loadScript } from '../../lib/load-script'
 import { getCDN } from '../../lib/parse-cdn'
+import {
+  applyDestinationMiddleware,
+  DestinationMiddlewareFunction,
+} from '../middleware'
+import { Context, ContextCancelation } from '../../core/context'
+import { Analytics } from '../../core/analytics'
+
+const klona = (evt: SegmentEvent): SegmentEvent =>
```

Comment:
I think `klona` also exists in the ajsDestinations plugin. Might be able to shave a few bytes if we consolidate them.